tests/unit/app/components/client_portal_modules.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import ClientPortalModules from 'app/js/components/client_portal_modules/client_portal_modules'
+
+describe('<ClientPortalModules />', () => {
+  it('renders with an empty list of modules', () => {
+    const modules = []
+    const component = shallow(<ClientPortalModules modules={modules} />)
+    expect(component.find('a').length).toBe(0)
+  })
+
+  it('renders with a single module', () => {
```

Comment:
Hmmmm, fair point... In this case doesn't make a lot of sense actually. This is just me biased by the behaviour of React Component's `this.props.children`, that is not the same with a single element or more than one. But, as I said, in this case it's true that maybe it's redundant.